api/src/wfl/service/slack.clj

```diff
@@ -82,8 +89,15 @@
   [queue]
   (if (seq queue)
     (let [{:keys [channel message]} (peek queue)]
-      (post-message channel message)
-      (pop queue))
+      (try
+        (post-message-and-throw-if-failure channel message)
+        (pop queue)
+        (catch Throwable t
+          (log/error "Failed when posting Slack message"
+                     :channel   channel
+                     :message   message
+                     :throwable t)
+          queue)))
```

Comment:
Hm.  If we fail to post a message, it stays in the queue, possibly blocking other messages if the issue isn't transient.

Reply:
Clever.  More ops than I'd normally be comfortable with in a catch, but if we got here, we had to be able to push it to the queue successfully to begin wiht.